merino-adm/src/remote_settings/reqwest_client.rs

```diff
@@ -20,10 +20,13 @@ pub struct ReqwestClient {
 
 impl ReqwestClient {
     /// Instantiate a new Reqwest client to perform HTTP requests.
-    pub fn new() -> ReqwestClient {
-        Self {
-            reqwest_client: reqwest::Client::new(),
```

Comment:
Just noticed that `request::Client::new()` could panic in certain cases, but the builders let you capture those errors without panicking.